storage/retry_conformance_test.go

```diff
@@ -199,10 +202,39 @@ var methods = map[string][]retryFunc{
 			if err != nil {
 				return err
 			}
-			_, err = io.Copy(ioutil.Discard, r)
+			wr, err := io.Copy(ioutil.Discard, r)
+			if got, want := wr, len(randomBytesToWrite); got != int64(want) {
+				return fmt.Errorf("body length mismatch\ngot:\n%v\n\nwant:\n%v", got, want)
+			}
 			return err
 		},
 	},
+	"storage.objects.download": {
+		func(ctx context.Context, c *Client, fs *resources, _ bool) error {
+			// Before running the test method, populate a large test object of 9 MiB.
```

Comment:
Since this happens before the test, I think this should be another fixture in `*resources` rather than something happening in the download method. We could call it LARGE_OBJECT. Does that seem reasonable? I understand that this would be a proto change also...

Reply:
Thanks Chris for the review. Your proposal is reasonable. I have reservations at this moment as it calls for changes in the json test schema, test proto, and moreover the implementations completed in other languages. 

Currently, Java uses a switch case bytes supplier that accommodates S8 large objects, Node.js has a similar helper function that generates given size objects, and Python uses a pytest fixture. My bad for not clarifying the construction details in the design doc earlier. How about we add this in the future improvements section?

Reply:
Discussed offline; we think this is a good idea but will file an issue for the work across repos rather than block this PR on the change.